### PR TITLE
add recipe for libsndfile

### PR DESCRIPTION
**Pull request by hbredin**

### CODE REVIEW

File: recipes/libsndfile/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+test:
+  commands:
+    - test -f $PREFIX/lib/libsndfile.dylib  # [osx]
+    - test -f $PREFIX/lib/libsndfile.so  # [linux]
+
+about:
+    home: http://www.mega-nerd.com/libsndfile
+    license: LGPL-2.1
+    license_family: LGPL
+    license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (ericdill) [inline recipes/libsndfile/meta.yaml]: Please submit a PR to https://github.com/erikd/libsndfile/ including the LICENSE in that project and then add a comment to this recipe that links to that PR so we know when we can stop including the LICENSE here.

File: recipes/libsndfile/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    summary: "libsndfile - a C library for reading and writing files containing sampled sound (such as MS Windows WAV and the Apple/SGI AIFF format) through one standard library interface"
```
Comment (ericdill) [inline recipes/libsndfile/meta.yaml]: Please add 
```yaml
dev_url: https://github.com/erikd/libsndfile
```

File: recipes/libsndfile/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{% set name = "libsndfile" %}
+{% set version = "1.0.27" %}
+{% set md5 = "fd1d97c6077f03b5d984d7956ffedb7a" %}
```
Comment (ericdill) [inline recipes/libsndfile/meta.yaml]: conda-forge prefers sha256. In this case `a391952f27f4a92ceb2b4c06493ac107896ed6c76be9a613a4731f076d30fac0`

File: recipes/libsndfile/meta.yaml (hunk 1 above)
Comment (hbredin) [inline recipes/libsndfile/meta.yaml]: Not sure I understand what you want me to put in the PR to `libsndfile` repo. 
Can you please be more specific?

FYI, all I did was copying https://github.com/erikd/libsndfile/blob/master/COPYING into LICENSE.

Comment (ericdill) [inline recipes/libsndfile/meta.yaml]: Oh, the license is already in the project. Great! Change the license_file line to: `license_file: COPYING` and you don't need to bundle LICENSE with this recipe.

**Comment by ericdill** [comment]
Oh come on github... there are no conflicts here :man_facepalming:

Can you rebase this and the argtable PR onto master?

**Pull request merged**